cloud/scope/common.go

```diff
@@ -1,21 +1,94 @@
 package scope
 
 import (
+	"fmt"
 	"net/http"
+	"sync"
 
+	"github.com/bxcodec/gotcha"
+	"github.com/bxcodec/gotcha/cache"
+	"github.com/bxcodec/httpcache"
+	"github.com/bxcodec/httpcache/cache/inmem"
+	"github.com/linode/cluster-api-provider-linode/util"
```

Comment:
Should we have the defaults in the util package if they are only being used here?